api/src/main/java/org/openmrs/hl7/handler/ORUR01Handler.java

```diff
@@ -359,8 +359,6 @@ private Message processORU_R01(ORU_R01 oru) throws HL7Exception {
 				finally {
 					// Handle obs-level exceptions
 					if (errorInHL7Queue != null) {
-						throw new HL7Exception(Context.getMessageSourceService().getMessage(
-						    "ORUR01.error.improperlyFormattedOBX",
 						    new Object[] { PipeParser.encode(obx, new EncodingCharacters('|', "^~\\&")) }, null),
```

Comment:
Hii @Devanshk9, line 326 to 364 is a 1 single statement, you cannot remove just some part of it. In this issue you not only need to remove this deprecated code but also replace it with a newer version of it.

**HINT**: Replace the current `HL7Exception(String message, ErrorCode error, Throwable cause)` with newer `HL7Exception(String message, ErrorCode error, Throwable cause)`

Reply:
hi @PrathameshMutkure thanks for help do i need to remove the 326 to 364 line

Reply:
Hi @PrathameshMutkure I hope now it is fine.

Reply:
Sorry for confusion, its not related to deprecated code but rather coding practices